src/test/kotlin/net/atos/zac/app/planitems/PlanItemsRESTServiceTest.kt

```diff
@@ -217,4 +230,38 @@ class PlanItemsRESTServiceTest : BehaviorSpec({
             }
         }
     }
+
+    Given("Zaak exists") {
+        val zaak = createZaak()
+
+        val mailGegevens = createMailGegevens()
+        val resultaat = Resultaat()
+
+        every { zrcClientService.readZaak(zaak.uuid) } returns zaak
+        every { policyService.readZaakRechten(zaak) } returns createZaakRechtenAllDeny(startenTaak = true)
+        every { policyService.checkZaakAfsluitbaar(zaak) } just runs
+        every { brcClientService.listBesluiten(zaak) } returns Optional.of(listOf(Besluit()))
+        every { zrcClientService.readResultaat(zaak.resultaat) } returns resultaat
+        every { zrcClientService.updateResultaat(any<Resultaat>()) } returns resultaat
+        every { mailService.sendMail(mailGegevens, any<Bronnen>()) } returns mailGegevens.body
+
+        When("Zaak is requested to close and send mail") {
```

Comment:
```suggestion
        When("A user event to settle the zaak and send a corresponding email is planned") {
```

? bit hard to describe it but something like that?

Reply:
Better :) Thanks!

Committed your suggestion